src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppECSFargate.recipe

```diff
@@ -45,25 +45,6 @@
                 }
             ]
         },
-
-        {
-            "Tests": [
-                {
-                    "Type": "FileExists",
-                    "Condition": {
-                        "FileName": "Dockerfile"
-                    }
-                }
-            ],
-            "Effect": {
-                "Fail": {
-                    "PriorityAdjustment": -55,
-                    "Include": true
-                }
-            }
-        },
-
```

Comment:
why are we no longer penalizing the lack of a Dockerfile?

Reply:
Because I switch the logic to raise the priority on Beanstalk recipe if there is no Dockerfile. That way the logic is just in the Beanstalk recipe instead of in all container recipes.